### PR TITLE
add $Identifier$ and cacheClause

### PR DESCRIPTION
**Pull request by happygod-lyc**
add $Identifier$ and cacheClause

### CODE REVIEW

**Comment by KvanTTT** [comment]
Could you please a sample that covers suggested changes or extend existing in https://github.com/antlr/grammars-v4/tree/master/sql/hive/v2/examples ?

File: sql/hive/v2/HiveParser.g4 [hunk 1]
```diff
@@ -1225,7 +1225,8 @@ The valuesClause rule below ensures that the parse tree for
 very similar to the tree for "insert into table FOO select a,b from BAR".
 */
 regularBody
-    : insertClause ( selectStatement | valuesClause)
+//    : insertClause ( selectStatement | valuesClause)
```
Comment (KvanTTT) [inline sql/hive/v2/HiveParser.g4:1228]: Please remove comment. Git keeps history well.

File: sql/hive/v2/HiveParser.g4 [hunk 2]
```diff
@@ -1225,7 +1225,8 @@ The valuesClause rule below ensures that the parse tree for
...
+    : (insertClause | cacheClause) ( selectStatement | valuesClause)
```
Comment (KvanTTT) [inline sql/hive/v2/HiveParser.g4:1229]: Redundant whitespace:

```suggestion
    : (insertClause | cacheClause) (selectStatement | valuesClause)
```

Comment (happygod-lyc) [reply sql/hive/v2/HiveParser.g4:1229]: ok

**Comment by KvanTTT** [comment]
There are some errors after your fixes on CI: https://github.com/antlr/grammars-v4/runs/6079660255?check_suite_focus=true#step:17:55 Please fix.

**Comment by teverett** [comment]
@happygod-lyc thanks for this PR.  Are you planning to fix the CI issues?

**Comment by teverett** [comment]
@happygod-lyc  I am closing this for now.  You are welcome to re-open it with merge fixes.

**Comment by happygod-lyc** [comment]
自动回复：来信收到

**Pull request closed without merging**